git/githistory/rewriter.go

```diff
@@ -253,6 +267,13 @@ func (r *Rewriter) Rewrite(opt *RewriteOptions) ([]byte, error) {
 			if err != nil {
 				return nil, err
 			}
+			if len(opt.ObjectMapFilePath) > 0 {
```

Comment:
Could this conditional be changed to:

```go
if objectMapFile != nil {
        // ...
}
```

This way, we:

  1. Minimize referencing the options type, which has a tendency to pollute the surrounding code-base, and

  2. Have a stronger guarantee that this block will not attempt to dereference a nil-pointer, should it become the case that `len(opt.ObjectMapFilePath) > 0 =/=> objectMapFile != nil`.

Reply:
Very good points, I have done the change